### PR TITLE
LPS-178776 | Add CanChangePaginationInAdminPage Poshi test

### PR DESCRIPTION
**Pull request by alexmds2004**
Description:
Add automation test about datasets

Plan:

Given create 6 data sets
//using API or via UI
When changing the page to 4 entries
Then confirm that only 4 entries are displayed 
When to switch to page 2 
Then 2 entries are displayed.

JIRA Ticket:
https://issues.liferay.com/browse/LPS-178776

### CODE REVIEW

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:frontend

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase [hunk 1]
```diff
@@ -30,12 +30,39 @@ definition {
...
 	test CanChangePaginationInAdminPage {
+		task ("Given create 6 data sets") {
+			for (var listDataSets : list "1,2,3,4,5,6") {
+				LexiconEntry.gotoAdd();
+
+				PortletEntry.inputName(name = "Test Dataset");
+
+				FrontendDataSetAdmin.selectProvider(key_type = "Channel");
+
+				PortletEntry.save();
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase:43]: Hey @alexmds2004 Could you create a macro for create a dataset and call it 6 times?

**Review by mateusralv — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase (hunk 1 above)
Comment (alexmds2004) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/FrontendDataSetViews.testcase:43]: Yes

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:help

**Comment by mateusralv** [comment]
ci:test:frontend-data-set

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:frontend-data-set

**Comment by mateusralv** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Review by mateusralv — APPROVED** [review]
GOOD JOB!!!

**Comment by mateusralv** [comment]
ci:forward

**Pull request closed without merging**